cookbook/setup_data/add_translation_packs.rst

```diff
@@ -0,0 +1,25 @@
+
+How to Add Translation Packs
+============================
+
```

Comment:
maybe some duplicate content with http://docs.akeneo.com/latest/contributing/translate.html contributing translate could contain a link to this new (and better) content.